mentorship_guidelines.md

```diff
@@ -0,0 +1,153 @@
+
+# Galaxy Mentorship Network Guidelines
+
+Welcome to the Galaxy Community Mentorship Program designed to help community members who may be bioinformaticians, data analysts, students, programmers, etc., become more proficient in the use of Galaxy tools and integration into the Galaxy Community.
+
+## Program Duration
+
+A typical mentorship program is scheduled to run for 8 weeks. Within this 8-week period, GMentors will work closely with GMentees and interact with other GMentors in the mentor community chat.
+
+Highlights of this mentorship program include:
+
+- Week 1 - an introductory call between GMentees, GMentors, and Network Managers.
+- Week 4 - mid-program feedback
+- Week 8 - final feedback.
+
+## Program Tracking
+
+### How we track progress
+
+- Two feedbacks during the duration of the program: week 4 and week 8
+- Meeting with the Network Manager 
+
+### What are we tracking?
+
+- Progress of GMentees’ project
+- Relationship between GMentor and GMentee(s)
+- Challenges
+
+
+There are three major roles in this mentorship program. They are:
+
+- Mentees
+- Mentors
+- Network Managers
+
+## Mentees
+
+Mentees are members of the Galaxy Community who:
+
+- Are new to the community and help to become proficient in the use of Galaxy tools.
+- Want to learn how to contribute efficiently.
+- Want to transition from one part of the Galaxy ecosystem to another.
+- Need to learn how to be a mentor.
+- Need mentorship in carrying out their projects.
+
+
+### First Steps: Getting Started
```

Comment:
I would move this section just before `Mentors`

Reply:
Do you mean the first steps sections?

Reply:
Yes

Reply:
Okay...all three roles have the same subsections starting with the first step(getting started) subsection. Moving the mentees' own down just before the `mentors` would require me to move that of the other roles too....to maintain consistency. Would that be okay?

Reply:
I thouht the same (see comments below) :smile:

Reply:
Great..I'll get to it now. Thank you! :)